cfgov/ask_cfpb/models/pages.py

```diff
@@ -67,6 +67,24 @@ def get_ask_breadcrumbs(category=None):
     return breadcrumbs
 
 
+def validate_page_number(request, paginator):
+    """
+    A utility for parsing a pagination request,
+    catching invalid page numbers and always returning
+    a valid page number, defaulting to 1.
+    """
+    raw_page = request.GET.get('page', 1)
+    try:
+        page_number = int(raw_page)
+    except ValueError:
+        page_number = 1
+    try:
+        paginator.page(page_number)
```

Comment:
Several places below you have this pattern:

```python
page_number = validate_page_number(request, paginator)
page = paginator.page(page_number)
```

It looks like everywhere we call `validate_page_number()` we're also calling `paginator.page()`. Would it make sense to simply have this function return `(page_number, page)`, and perhaps rename it something like `get_valid_page`? I don't know how expensive calls to `paginator.page` might be but that would reduce them, except in the case of an `InvalidPage` where we'd have to call it again with `1`.